_includes/tutorials/rekeying-function/ksql/markup/dev/copy-udf-jar.adoc

```diff
@@ -0,0 +1,5 @@
+Create a directory for the KSQL extensions and copy the new jar file into this location.  We will refer to this location in the `docker-compose.yml` file used to launch the Confluent Platform.
```

Comment:
Would it be possible to set things up so that this extra step doesn't need to be taken? The thing I have in mind is that you may want to iterate on your function as you use KSQL. You'd have to reboot KSQL to pick up the new jar, but at least you wouldn't need to keep copying from target to ext everytime you uberjar.

Reply:
Also good to note in the text what the reload policy is :)

Reply:
Done